app/pages/vaccinations/context.test.js

```diff
@@ -0,0 +1,103 @@
+import { createVaccinationsSolutionListPageContext } from './context';
+
+import manifest from './manifest.json';
+
+const defaultExpectedContext = {
+  title: manifest.title,
+  pageDescription: manifest.description,
+  insetText: manifest.insetText,
+  backLinkPath: '/',
+  solutions: [],
+};
+
+describe('createVaccinationsSolutionListPageContext - vaccinations', () => {
+  it('should create a context for the vaccinations page with one solution', () => {
+    const expectedContext = {
+      ...defaultExpectedContext,
+      solutions: [
+        {
+          id: '00001',
+          name: 'The first solution',
+          summary: 'Some solution summary',
+          supplierName: 'Some supplier',
+          vaccinations: {
+            title: 'Some title',
+            list: [
+              'First list item',
+              'Second list item',
+              'Second list item',
+            ],
+          },
+          viewSolutionUrl: '/solutions/vaccinations/00001',
+        },
+      ],
+    };
+
+    const data = {
+      solutions: [
+        {
+          id: '00001',
+          name: 'The first solution',
+          summary: 'Some solution summary',
+          supplier: {
+            id: '1',
+            name: 'Some supplier',
+          },
+          vaccinations: {
+            title: 'Some title',
+            list: [
+              'First list item',
+              'Second list item',
+              'Second list item',
+            ],
```

Comment:
Could we extract the common properties/values (everything except `supplierName`, `viewSolutionUrl` and `supplier`?) to another (solution) object and use the spread operator in the `expectedContext` and `data` objects instead here?